### DISCUSSION TITLE
Query: documentation  wiki/GUI:-Sound

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1318] [General]
As before, I was just now looking to update the helptext for the Sound panel, and found these inconsistencies when referencing the wiki content ;

Image of GUI panel does not show `System default` or `Swap channels`  items, and descriptions for these settings are missing.

Image of GUI shows only Paula volume slider available, and helptext states other options here are unavailable, however with the latest amiberry releases, all 4 volume options appear available. (I think more than just Paula works now?)

Could you at your convenience, update the wiki in these regards please? I can then update the helptext accordingly.

TIA

**Comment by midwan** [comment]
Wiki updated: https://github.com/BlitterStudio/amiberry/wiki/GUI:-Sound
I've also updated the help text accordingly.

**Comment by giantclambake** [comment]
Thanks, much improved ;)
